docs/atlascli/command/atlas-projects-create.txt

```diff
@@ -75,6 +75,10 @@ Options
      - string
      - false
      - Unique 24-digit string that identifies the Atlas user to be granted the Project Owner role on the specified project. If unspecified, this value defaults to the user ID of the oldest Organization Owner.
+   * - --tag
+     - stringToString
```

Comment:
for @mongodb/docs-cloud-team I think we should update https://github.com/mongodb-labs/cobra2snooty/blob/main/flags.go to better handle these

Reply:
Agreed. In the meantime, is there a way we can prevent this for now? Would `StringVar` not work in this scenario?

Reply:
`StringVar` will not work in this scenario since we accept multiple tags.

`atlas project create MyProject --tag environment=testing --tag prod=false` is a valid command.
With `StringVar` you would only be able to provide one tag.